docs/communication/fearless-feedback.md

```diff
@@ -0,0 +1,94 @@
+# Fearless feedback
+
+Fearless feedback is feedback which is direct, honest, timely, brief, constructive, and provided with good intention.
+
+Many people struggle with giving or receiving feedback, which is normal. Through relfex people become defensive or offended, espeically if the feedback is not delivered well.
+
+Feedback, however, is integral to growth and confidence - through celebrating achievements, validating actions or opinions, and learning. It provides a mutual understanding of a situation, action, or emotion, to allow for easy and respectful communication.
+
+## Feedback matrix
+
+There are four different types of feedback that exist on this spectrum.
+
+|             |              |              |
+| :---------- | :----------: | -----------: |
+|             | **Positive** |
+| **General** |      +       | **Specific** |
+|             | **Negative** |              |
+
+**Examples of each type of feedback:**
+
+> Positive, general: "_Harold, you did a great job the other day!_"
+>
+> Negative, general: "_Emerald, I saw that PR you put through. It was really bad._"
+>
+> Positive, specific: "_Harold, you did a great presentation on Wednesday, you were so confident. I learned a lot about how to use GitHub._"
+>
+> Negative, specific: "_Emerald, I saw PR #70, some of the naming conventions were did not follow the contribution guide, making it hard to understand._"
+
+Key takeaways:
+
+- Negative feedback should always be specific
+- General feedback for positive feedback is okay, but not ideal
+- Specific feedback is the most impactful and the most productive
+- Specifics are the best way to ensure that feedback is taken on-board and helpful
+
+## Feedback delivery
```

Comment:
**suggestion:** It would be great to add a section about receiving feedback. What do you think?